drupal-org.make

```diff
@@ -1,7 +1,7 @@
 api: '2'
 core: 7.x
 includes:
-  - "https://raw.githubusercontent.com/NuCivic/visualization_entity/7.x-1.0-beta1/visualization_entity.make"
+  - "https://raw.githubusercontent.com/NuCivic/visualization_entity/7.x-1.x/visualization_entity.make"
```

Comment:
@rhabbachi you need to point the actual module downloaded in the makefile to the same tag or branch as the include; we don't want the beta-1 code but the 7.x-1.x makefile